hack/dockerized

```diff
@@ -44,7 +44,7 @@ _rsync() {
 rm ${KUBEVIRT_DIR}/.glide*.hash -f
 
 # Copy kubevirt into the persistent docker volume
-_rsync --delete --exclude 'cluster/vagrant*' --exclude "_out" --exclude ".vagrant" ${KUBEVIRT_DIR}/ "rsync://root@127.0.0.1:${RSYNCD_PORT}/build"
+_rsync --delete --exclude 'cluster/**/.kubectl' --exclude 'cluster/**/.oc' --exclude 'cluster/**/.kubeconfig' --exclude "_out" --exclude ".vagrant" ${KUBEVIRT_DIR}/ "rsync://root@127.0.0.1:${RSYNCD_PORT}/build"
```

Comment:
what's the context here for why some files in cluster should be synced.

Reply:
The reason is pretty dump: The build step also formats all code, including bash scripts. If we don't pass in all bash scripts in there they don't get formatted.

Reply:
I hope we will merge it as soon as possible, it just too annoying to checkout bash scripts after each `make` command.